Source/core/NodeId.h

```diff
@@ -46,9 +46,11 @@
 #endif
 
 #ifdef __CORE_BLUETOOTH_SUPPORT__
-#include <../include/bluetooth/bluetooth.h>
-#include <../include/bluetooth/hci.h>
-#include <../include/bluetooth/l2cap.h>
+PUSH_WARNING(DISABLE_WARNING_PEDANTIC)
+#include "bluez/bluetooth.h"
```

Comment:
I'm not a hippie to have feelings, but naming the folder bluez feels like Thunder is being dependent on bluez but we're only dependent on the bluetooth SIG-defined magic numbers / structures only.

Reply:
Correct, but still there is a include path, so build time there is a dependency. By naming it bluez at least it's clear who's API we expect.